pkg/backoff/exponential.go

```diff
@@ -57,3 +59,7 @@ func (b *ExponentialBackOff) NextBackOff() time.Duration {
 	}
 	return duration
 }
+
+func (b *ExponentialBackOff) Reset() {
```

Comment:
On L58 (can't comment there) shouldn't we do `return b.MaxDuration` not infinite?

I think our expectation is if we have max=30s then backoffs are:

0s, 1s, 2s, 4s, 8s, 16s, 30s, 30s, 30s, 30s

not

0s, 1s, 2s, 4s, 8s, 16s, 30s, infinity

?

Reply:
There are two fields MaxInterval vs MaxElapsedTime

If MaxElapsedTime = 0, we will return 0s, 1s, 2s, 4s, 8s, 16s, 30s, 30s, 30s, 30s,30.........

Reply:
I see. thanks